Providers/AppsFlyerProvider.m

```diff
@@ -33,7 +33,11 @@ - (void)identifyUserWithID:(NSString *)userID andEmailAddress:(NSString *)email
 }
 
 - (void)event:(NSString *)event withProperties:(NSDictionary *)properties {
-    [[AppsFlyerTracker sharedTracker] trackEvent:event withValue:@""];
+    if (properties != nil) {
+        [[AppsFlyerTracker sharedTracker] trackEvent:event withValues:properties];
```

Comment:
Because I wasn't sure if AppsFlyerTracker handles nil properly (I thought this might have been the reason to do this in the first place) - and there is no access to the code to make sure..